src/features/Bluetooth/Download/DownloadManager.ts

```diff
@@ -25,15 +25,15 @@ export class DownloadManager {
   // point. Where the starting point is the timestamp for the next log.
   calculateNumberOfLogsToSave = (
     nextPossibleLogTime = 0,
-    logInterval: number,
-    timeNow = moment().unix()
+    logInterval: UnixTimestamp,
```

Comment:
Just changed this all to be using utilService and unix timestamps rather than moment.

The test coverage here is pretty solid, so confident this is OK (Going to regret that?)